src/sentry/integrations/jira/integration.py

```diff
@@ -958,6 +961,48 @@ def get_resolve_sync_action(self, data: Mapping[str, Any]) -> ResolveSyncAction:
             should_unresolve=c_from in done_statuses and c_to not in done_statuses,
         )
 
+    def migrate_issues(self):
+        for project in Project.objects.filter(organization_id=self.organization_id):
+            plugin = None
+            for p in plugins.for_project(project):
+                if self.model.provider.startswith(p.slug) and p.get_option(
```

Comment:
This makes it work for `jira` and `jira_server` as provider strings when the plugin slug is always `jira`.